_posts/2016-10-31-Kickstarters-Project.md

```diff
@@ -0,0 +1,205 @@
+And a lot of great food ofcourse.
+
+
+
+## August - Learning
+
+The focus during the first month of the Kickstarters project was primarily on learning.
+The first few days the kickstarters had to improve their softskills by learning how to be more assertive towards the client when needed.
+They also learned to introduce themselves properly with the emphasis on their qualities and strengths, to ensure they make a good first impression themselves when going to the client.
+
+The kickstarters were kept up to date with the preferred technologies, editors and best practices used by JWorks.
+During the first month they received different courses in which they could improve their technical skills about:
+
+ * Back-end
+    * Java (7 & 8) + JavaEE
+    * Spring
+    * JPA
+    * Webservices (REST & SOAP)
+    * MongoDB
+
+ * Front-end
+    * HTML & CSS
+    * Javascript & Typescript
+    * Angular
+    * Ionic
+
+These courses were given by the JWorks unit who tried to teach the kickstarters a much as possible with theoretical material and some exercises afterwards.
+Unfortunately these technical skills aren't enough to survive in the forever changing IT world.
+This is why some extra help was provided in the shape of books and courses about how to write clean code, how to work agile and learning how to work in a team while understanding and using the SCRUM principles.
+
+So by paying attention to technical development with the necessary certification processes and also focussing on the development of soft skills like communication, advising and collaboration,
+Ordina commits to the personal development of these kickstarters.
+
+
+
+## September - Dev-case
+
+The focus during the second month was on the implementation of this year's dev-case.
+Although they still had to follow a few courses along the way, like the basic priciples of security, the use of GIT and learning how to use MongoDB.
+
+### SensyMcSenseFace was born
+-- <cite>Chosen by popular vote, who would have guessed it...</cite></br>
+
+The kickstarters already had learned how to write clean code and how to do this in the best possible way.
+The purpose of the SensyMcSenseFace project was to give the kickstarters a use case where they could develop an end-to-end IoT solution, in which they could test and use their newly acquired skills.
+
+<p style="text-align: center;">
+  <img alt="SensyMcSenseFaceProject" src="/img/IoT.jpg">
+</p>
+
+### What did the client want
+
+The kickstarters were supposed to build an application that accepts incoming data, while being able to process this data and output it in a more user friendly way.
+The data would be send by 3 different sensors.
+
+The sensors that are used:
+   * Temperature sensor
+   * Humidity sensor
+   * Motion sensor
+
+These sensors sends some data every few seconds to the back-end, where this data is being processed and send back to the front-end.
```

Comment:
These sensors **send** some data every few seconds to the back end. The back end then processes this data and sends it back to the front end.